packages/match-media/src/useBreakpointIndex.js

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect, useCallback } from 'react'
+import { useThemeUI } from 'theme-ui'
+
+const useBreakpointIndex = () => {
+  const {
+    theme: { breakpoints },
```

Comment:
Also wonder if providing a default value for `breakpoints` would be good here since `@styled-system/css` will fall back to defaults

Reply:
Yes actually I was wondering about that, should we reference it from here https://github.com/styled-system/styled-system/blob/master/packages/css/src/index.js#L10 or just embed it.

Reply:
I think it's fairly harmless to hard-code the values in this case